docs/external-listener/index.md

```diff
@@ -68,6 +68,19 @@ To configure an external listener that uses the LoadBalancer access method, comp
 
     - To use Envoy, set the `ingressController` field in the `KafkaCluster` custom resource to `envoy`. For an example, [see](https://github.com/banzaicloud/koperator/blob/672b19d49e5c0a22f9658181003beddb56f17d33/config/samples/banzaicloud_v1beta1_kafkacluster.yaml#L12).
 
+      For OpenShift:
+
+      ```yaml
+      spec:
+        envoyConfig:
```

Comment:
I would put these settings between ...
spec:
...
envoyconfig..
...

Reply:
Fixed in https://github.com/banzaicloud/koperator-docs/compare/35bddf22181567081bf9b6f094806a38ab72c367..42fff0bdad627e7158f9a7a1a41dacaaeefc2ac5.